packages/common/src/api/collection.ts

```diff
@@ -28,7 +28,7 @@ const collectionApi = createApi({
         }: { playlistId: Nullable<ID>; currentUserId?: Nullable<ID> },
         { audiusSdk }
       ) => {
-        if (!playlistId) return null
+        if (!playlistId || playlistId === -1) return null
```

Comment:
nit: Might as well do something like `id > 0` ? We don't use a constant or anything to mark -1 as special, so we should probably ensure that the values we're using in these cases are valid values.